checkstyle.xml

```diff
@@ -16,196 +16,215 @@
         </module>
         <module name="InterfaceTypeParameterName">
-            <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)" />
+            <property name="format" value="(^[A-Z][0-9]?)$|([A-Z][a-zA-Z0-9]*[T]$)"/>
             <message key="name.invalidPattern"
-                     value="Interface type name ''{0}'' must match pattern ''{1}''." />
+                     value="Interface type name ''{0}'' must match pattern ''{1}''."/>
         </module>
-        <module name="NoFinalizer" />
+        <module name="NoFinalizer"/>
         <module name="GenericWhitespace">
             <message key="ws.followed"
-                     value="GenericWhitespace ''{0}'' is followed by whitespace." />
+                     value="GenericWhitespace ''{0}'' is followed by whitespace."/>
             <message key="ws.preceded"
-                     value="GenericWhitespace ''{0}'' is preceded with whitespace." />
+                     value="GenericWhitespace ''{0}'' is preceded with whitespace."/>
             <message key="ws.illegalFollow"
-                     value="GenericWhitespace ''{0}'' should followed by whitespace." />
+                     value="GenericWhitespace ''{0}'' should followed by whitespace."/>
             <message key="ws.notPreceded"
-                     value="GenericWhitespace ''{0}'' is not preceded with whitespace." />
+                     value="GenericWhitespace ''{0}'' is not preceded with whitespace."/>
         </module>
         <module name="Indentation">
-            <property name="basicOffset" value="4" />
-            <property name="braceAdjustment" value="0" />
-            <property name="caseIndent" value="4" />
-            <property name="throwsIndent" value="4" />
-            <property name="lineWrappingIndentation" value="4" />
-            <property name="arrayInitIndent" value="4" />
+            <property name="basicOffset" value="2"/>
+            <property name="braceAdjustment" value="0"/>
+            <property name="caseIndent" value="2"/>
+            <property name="throwsIndent" value="4"/>
+            <property name="lineWrappingIndentation" value="4"/>
+            <property name="arrayInitIndent" value="2"/>
         </module>
         <module name="AbbreviationAsWordInName">
-            <property name="ignoreFinal" value="true" />
-            <property name="allowedAbbreviationLength" value="4" /> <!-- Allow 'GRIP' in all caps -->
+            <property name="ignoreFinal" value="true"/>
+            <property name="allowedAbbreviationLength" value="4"/> <!-- Allow 'GRIP' in all caps -->
         </module>
-        <module name="OverloadMethodsDeclarationOrder" />
+        <module name="OverloadMethodsDeclarationOrder"/>
         <module name="VariableDeclarationUsageDistance">
-            <property name="allowedDistance" value="5" />
-        </module>
-        <module name="AvoidStarImport" />
-        <module name="CustomImportOrder">
```

Comment:
It's possible for someone to set up their IDE (Eclipse, Intellij, Netbeans, whatever) to have an import order that's not the same as someone else's IDE. Setting an explicit format in checkstyle ensures that nobody will accidentally reformat imports and needlessly change lines